e2e-tests/tfm/cypress/e2e/pages/caseDealPage.js

```diff
@@ -10,6 +10,7 @@ const caseDealPage = {
   bankSecurityFacilityText: () => cy.get('[data-cy="bank-security-facility-text"]'),
   cancelDealButton: () => cy.get('[data-cy="cancel-deal-button"]'),
   dealStage: () => cy.get('[data-cy="ukef-deal-stage-value"]'),
+  successBanner: () => cy.get('[data-cy="success-banner"]'),
```

Comment:
Are we able to move `successBanner` into `partials` please? I can only see one other instance of `successBanner` in `unissuedFacilityTable`, so could remove that as well

Reply:
I can do, which `partials` file do you expect it to live in? `unissuedFacilityTable` is in GEF as opposed to TFM-UI so we'd have to make sure it's a partial file that both services use. Possibly we need a new file, but then it would just have `successBanner` in it and nothing else, and not sure what to call it - what do you think? 

![image](https://github.com/user-attachments/assets/0e72b249-b681-490a-8f9b-a6758737c133)

Reply:
I was just thinking in the index, where we have backLink, cancelLink, errorSummary etc 

```
e2e-tests/tfm/cypress/e2e/partials/index.js
```

Reply:
Got it, good idea thanks. I've moved that over. I won't change the other instances in this PR in `unissuedFacilityTable` as that's used in a lot of touched files so will show up as a big diff in this PR.